packages/firestore/test/unit/remote/bloom_filter.test.ts

```diff
@@ -198,25 +198,34 @@ describe('BloomFilter', () => {
         TEST_DATA.count5000Rate0001TestResult
       );
     });
-    it('mightContain result for 50000 documents with 1 false positive rate', () => {
+
+    // Skip the large sized golden tests below as they are flaky on CI.
```

Comment:
Please update these comment to _also_ explain _why_ they are flaky in CI. Actually, I'd just omit the wording about "flaky" and just say that they are very slow and slow down unit test runs without adding much additional coverage compared to the other golden tests.